package/network/services/hostapd/Makefile

```diff
@@ -239,6 +240,7 @@ define Package/wpad
 $(call Package/wpad/Default,$(1))
   TITLE+= (built-in full)
   VARIANT:=wpad-full-internal
+  DEPENDS+=+libpcsclite
```

Comment:
With this change all the hostapd variants would depend on the libpcsclite library which increases the image size. I would prefer if this only gets added to the full hostapd or wpa_supplicant versions.

Reply:
My bad, I thought that was the define for the full version (since there is a basic one). Which should I edit?

If I'm correct, the full config affect almost all variants (OpenSSL, wolfssl, full ...)